build.toml

```diff
@@ -13,12 +13,19 @@ target = 'My Computer'
 build_spec = 'Configuration Release'
 
 [[build.steps]]
-name = 'Engine Library'
+name = 'PharLap Engine Library'
```

Comment:
The engine library build steps can be combined into a single step instead of Pharlap and Linux. Then, if we ever do support any other OSes, we don't have to revisit this file. The only reason to leave it this way would be to easily disable the LinuxRT build, if you think that is something you might want to do.

name = 'Engine Libraries'
type = 'lvBuildSpecAllTargets'
project = '{cd}'
build_spec = 'Engine Release'